### PR TITLE
Disable document sent button during API activity

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Part of #1752

#### What's this PR do?
Disables the document sent button during API activity

#### How should this be manually tested?
Apply for financial aid, then click the button which submits the document sent date. It should show a spinner and be unclickable during that time.

### CODE REVIEW

**Comment by aliceriot** [comment]
functionality looks good! looking at code now.

File: static/js/components/dashboard/CourseAction.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ export default class CourseAction extends React.Component {
 
     return (
       <SpinnerButton
-        className="dashboard-button"
+        className="dashboard-button pay-button"
```
Comment (aliceriot) [inline static/js/components/dashboard/CourseAction.js]: are these changes related to rebasing over another change? Just wondering why there are changes to `CourseAction` in this pr.

File: static/js/components/dashboard/FinancialAidCard.js [hunk 2]
```diff
@@ -122,7 +132,7 @@ export default class FinancialAidCard extends React.Component {
       </div>
       <div className="pricing-actions">
         <button
-          className="mdl-button dashboard-button"
+          className="mdl-button dashboard-button calculate-cost-button"
```
Comment (aliceriot) [inline static/js/components/dashboard/FinancialAidCard.js]: was this added to disambiguate the buttons?

Comment (noisecapella) [inline static/js/components/dashboard/FinancialAidCard.js]: Yes, just for clarity

File: static/js/components/dashboard/CourseAction.js (hunk 1 above)
Comment (noisecapella) [inline static/js/components/dashboard/CourseAction.js]: I think here I just wanted to make the selector in the test more specific. There's no functional change going on here
